go.mod

```diff
@@ -4,11 +4,12 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5 // indirect
 	github.com/apex/log v1.9.0
-	github.com/buildpacks/imgutil v0.0.0-20201211223552-8581300fe2b2
+	github.com/buildpacks/imgutil v0.0.0-20210115182929-e2b7b1a5467a
 	github.com/containerd/containerd v1.3.3 // indirect
 	github.com/docker/cli v0.0.0-20200312141509-ef2f64abbd37 // indirect
 	github.com/docker/docker v1.4.2-0.20190924003213-a8608b5b67c7
 	github.com/golang/mock v1.4.4
+	github.com/golang/protobuf v1.4.2 // indirect
```

Comment:
Same comment as https://github.com/buildpacks/lifecycle/pull/498/files#r561087744.

Reply:
Going to remove it.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/721cf13f78654cd7978864642557e02ce6a9931d